config-service-change-event-api/src/main/proto/org/hypertrace/config/change/event/v1/config_change_event_value.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.config.change.event.v1;
+
+import "google/protobuf/struct.proto";
+
+message ConfigChangeEventValue {
+  oneof event {
+    ConfigCreateEvent create_event = 1;
+    ConfigUpdateEvent update_event = 2;
+    ConfigDeleteEvent delete_event = 3;
+  }
+  optional string user_id = 4;
+  optional string user_name = 5;
+}
+
+message ConfigCreateEvent {
+  string created_config = 1;
```

Comment:
since we're storing json in this string, can we clarify the name to `created_config_json` (and below)